pyproject.toml

```diff
@@ -117,7 +117,7 @@ PyJWT = "~1.7"
 python-dateutil = "~2.8"
 tensorflow = "~2.3"
 tensorflow_hub = "~0.8"
-tensorflow-addons = "~0.11"
+tensorflow-addons = ">= 0.10, <= 0.12"
```

Comment:
I think you have to run `poetry update` again as the hash should change for `poetry.lock`

Reply:
already on it :)